### ISSUE TITLE
SAR to Optical image generation

### ISSUE THREAD

**Issue by arrrrr3186** [junyanz/pytorch-CycleGAN-and-pix2pix#1626]
I am cycleGan Model for sar to optical image generation, results are so okay okay (not to bad), 

i want to know like how to improve optical images accuracy?

**Comment by gabaebae123** [comment]
Could you share the SAR-OPT data set？ @arrrrr3186

**Comment by Rjaat** [comment]
Hi @arrrrr3186 good to hear you got quite better results. Can you share the training hyperparameters please! also share the results if possible.
